api/app/serializers/concerns/abilities.rb

```diff
@@ -0,0 +1,68 @@
+# Includes serializer authorization
+module Abilities
```

Comment:
This is going to return results that I know will be false. For example, currently, an unauthenticated user can definitely `read` a comment, but this concern will return false for all abilities if the user isn't logged in (see `CommentAuthorizer.readable_by?`). I don't think returning false for an unauthenticated user is the right way to go... thoughts?